vtex-apps/order-placed
# Added extra props and CSS handles for styling

#### What is the purpose of this pull request?
The PR's purpose is to add extra CSS handles and props to deal with styling certain blocks, re-arranging them in a certain way (as per client's request), displaying extra info (with no impact on default implementations due to the fact that the new props have a default value)

#### What problem is this solving?
Helps provide a solution for a new VTEX client who requested a special design for the confirmation page

#### How should this be manually tested?

1. Link the changes on a workspace with the default order-placed template
2. Turn on some of the newly added props
3. See the magic happen :) 

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/71461884/110390944-6f2edf00-806f-11eb-97f6-224aba5053fc.png)

#### Types of changes
